### PR TITLE
#3076 Add POP Max

### LINKED ISSUE #3076: Pop
**Channel Name: POP
**Channel ID: Pop.uk
**Country: UK
**Language: English
**External Link (optional):
**Notes (optional): Free to air UK channel

### PR DESCRIPTION
**Pull request by UltraHDR**
Link doesn't work in VLC but works in other players

### CODE REVIEW

File: streams/uk.m3u [hunk 1]
```diff
@@ -329,6 +329,8 @@ http://45.76.186.114:8080/hls/noortv.m3u8
 https://live.smartmediahouse.tv:8443/hls/ch1.m3u8
 #EXTINF:-1 tvg-id="PBSAmerica.uk" status="error",PBS America (720p)
 https://uktv.vercel.app/pbsamerica.m3u8
+#EXTINF:-1 tvg-id="Pop.uk" status="online",Pop (576p)
+http://live-sonybebanjo.simplestreamcdn.com/live/popmax/bitrate1.isml/live.mpd
```
Comment (UltraHDR) [inline streams/uk.m3u:333]:
```suggestion
https://live-sonybebanjo.simplestreamcdn.com/live/popmax/bitrate1.isml/live.mpd
```

**Review by freearhey — DISMISSED** [review]

File: streams/uk.m3u [hunk 2]
```diff
@@ -329,6 +329,8 @@ http://45.76.186.114:8080/hls/noortv.m3u8
 https://live.smartmediahouse.tv:8443/hls/ch1.m3u8
 #EXTINF:-1 tvg-id="PBSAmerica.uk" status="error",PBS America (720p)
 https://uktv.vercel.app/pbsamerica.m3u8
+#EXTINF:-1 tvg-id="Pop.uk" status="online",Pop (576p)
```
Comment (UltraHDR) [inline streams/uk.m3u:332]:
```suggestion
#EXTINF:-1 tvg-id="PopMax.uk" status="online",POP Max (576p)
```

**Review by freearhey — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
https://live-sonybebanjo.simplestreamcdn.com/live/popmax/bitrate1.isml/live.m3u8 this should work for VLC.

File: streams/uk.m3u [hunk 3]
```diff
@@ -329,6 +329,8 @@ http://45.76.186.114:8080/hls/noortv.m3u8
 https://live.smartmediahouse.tv:8443/hls/ch1.m3u8
 #EXTINF:-1 tvg-id="PBSAmerica.uk" status="error",PBS America (720p)
 https://uktv.vercel.app/pbsamerica.m3u8
+#EXTINF:-1 tvg-id="PopMax.uk" status="online",POP Max (576p)
```
Comment (UltraHDR) [inline streams/uk.m3u:332]:
```suggestion
#EXTINF:-1 tvg-id="PopMax.uk" status="online",POP Max (576p)
https://live-sonybebanjo.simplestreamcdn.com/live/popmax/bitrate1.isml/live.m3u8
#EXTINF:-1 tvg-id="PopMax.uk" status="online",POP Max (576p)
```

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**